content/source/docs/enterprise/private/faq.html.md

```diff
@@ -219,6 +220,16 @@ When displaying Terraform Runs, Private Terraform Enterprise has JavaScript that
 * Private Terraform Enterprise uses the configured SMTP endpoint for sending emails
 * Twilio can optionally be set up for for SMS-based 2FA (virtual TOTP support is available separately which does not make external API calls)
 
+---
+## Disabling VCS webhook SSL verification
+
+If running PTFE with a self-signed certificate (which is not recommended) or with a certificate that the connected VCS system does not trust, webhooks on the VCS must have SSL verification disabled in order to push data to PTFE. This configuration is _not recommended_ but is included here for reference.
```

Comment:
I would also add "or add the self-signed certificate to the VCS server." (which appears to be how BBS handles this.